tests/core/full_node/test_full_node.py

```diff
@@ -388,8 +388,11 @@ async def do_test_block_compression(self, setup_two_nodes_and_wallet, empty_bloc
 
     @pytest.mark.asyncio
     async def test_block_compression(self, setup_two_nodes_and_wallet, empty_blockchain):
-        self.do_test_block_compression(setup_two_nodes_and_wallet, empty_blockchain, 10000, True)
-        self.do_test_block_compression(setup_two_nodes_and_wallet, empty_blockchain, 3000000000000, False)
+        await self.do_test_block_compression(setup_two_nodes_and_wallet, empty_blockchain, 10000, True)
+
+    @pytest.mark.asyncio
+    async def test_block_compression_2(self, setup_two_nodes_and_wallet, empty_blockchain):
+        await self.do_test_block_compression(setup_two_nodes_and_wallet, empty_blockchain, 3000000000000, False)
```

Comment:
The intention was that the reorg tests are not really affected by the `tx_size`, and they are very slow, since they try a bunch of combinations of scenarios, so only want to run them once.